src/Lean/Elab/BuiltinNotation.lean

```diff
@@ -290,7 +290,7 @@ where
   extra state, and return it. Otherwise, we just return `stx`.
   -/
   go : Syntax → StateT (Array Ident) MacroM Syntax
-  | stx@`(($(_))) => pure stx
+  | stx@`($_:hygienicLParen$(_))) => pure stx
```

Comment:
It scares me that this syntax no longer has matching parentheses, and will break bracket highlighting in vscode

Reply:
Why are you commenting on the pre-stage0 commit?

Reply:
The bootstrapping issues were... interesting to work out.

Something that didn't help is that the Parenthesizer makes use of a syntax quotation with parentheses without any of the term parsers in scope, so everything needed to deal with both `symbol "("` and `hygienicLParen` before the stage0 update (hence why `hygienicLParen` was marked as a pseudo-kind).